app/models/manageiq/providers/ansible_tower/configuration_manager/playbook.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook < ConfigurationScript
+  belongs_to  :configuration_project, class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationSource"
+  has_many    :configuration_script,  class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript"
```

Comment:
Also also, has_many configuration_scripts should be plural, but why is this has_many needed?  Is there a tree of playbooks?